accord-core/src/test/java/accord/impl/basic/Cluster.java

```diff
@@ -105,26 +148,17 @@ NodeSink create(Id self, RandomSource random)
         return sink;
     }
 
-    private void add(Packet packet)
+    void add(Packet packet, long delay, TimeUnit unit)
     {
         MessageType type = packet.message.type();
         if (type != null)
             statsMap.computeIfAbsent(type, ignore -> new Stats()).count++;
-        boolean isReply = packet.message instanceof Reply;
         if (trace.isTraceEnabled())
-            trace.trace("{} {} {}", clock++, isReply ? "RPLY" : "SEND", packet);
+            trace.trace("{} {} {}", clock++, packet.message instanceof Reply ? "RPLY" : "SEND", packet);
         if (lookup.apply(packet.dst) == null) responseSink.accept(packet);
-        else pending.add(packet);
-    }
+        else if (delay == 0)                  pending.add(packet);
```

Comment:
your rebase looks to be missing the recent commit where I dropped this branch; `pending.add` is impl as `add(packet, 0, nanos)` so no reason to special case this